systemtest/cmd/sendotlp/main.go

```diff
@@ -237,7 +245,15 @@ func newOTLPGRPCExporters(ctx context.Context, endpointURL *url.URL) (*otlpExpor
 		// If http:// is specified, then use insecure (plaintext).
 		transportCredentials = insecure.NewCredentials()
 	case "https":
-		transportCredentials = credentials.NewClientTLSFromCert(nil, "")
+		var cp *x509.CertPool
+		if *cacert != "" {
+			newCP, err := appendCert(*cacert)
```

Comment:
Nit: We could avoid using a buffer variable by explicitly declaring `err` first:

```suggestion
			var err error
			cp, err = appendCert(*cacert)
```